dind/README.md

```diff
@@ -0,0 +1,46 @@
+# rke DinD
+
+Docker-in-Docker (DinD) feature is created mainly for testing.
+
+DinD creates docker containers locally which are then used as cluster nodes.
+
+It still requires a cluster.yml with minimal information:
+
+```
+kubernetes_version: "$KUBE_VERSION"
```

Comment:
Either leave this out (as it is minimal) or specify an example so users know what to put here

Reply:
I wasted a lot of time trying to understand what needs to be passed to a cluster.yaml file of a dind deployment, so I'd prefer to leave it. I can specify how to find `$KUBE_VERSION`